test/envoye2e/env/setup.go

```diff
@@ -370,6 +381,21 @@ func (s *TestSetup) WaitServerEnvoyReady() {
 	s.WaitEnvoyReady(s.Ports().ServerAdminPort)
 }
 
+func checkStats(expectedStatsMap map[string]int, actualStatsMap map[string]int) error {
+
```

Comment:
nit: extra line?

Reply:
Removed this function.. not needed anymore